spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/convert/StructAccessor.java

```diff
@@ -99,6 +100,7 @@ public class StructAccessor {
           .build();
 
   // @formatter:on
+  private static final String EXCEPTION_NARRATIVE_COL_NOT_ARRAY = "Column is not an ARRAY type: ";
```

Comment:
This is very minor, but we don't have a constant naming convention with `NARRATIVE` in this code base, so might as well skip it and make the constant name shorter.